src/inference/src/dev/threading/cpu_streams_executor.cpp

```diff
@@ -120,7 +120,6 @@ struct CPUStreamsExecutor::Impl {
                 _impl->_streamIdQueue.push(_streamId);
             }
 #if OV_THREAD == OV_THREAD_TBB || OV_THREAD == OV_THREAD_TBB_AUTO
-            set_cpu_used(_cpu_ids, NOT_USED);
```

Comment:
If we don't reset cpu_map during Stream destructing, it will bring some problem for models loaded --> unloaded --> loaded cases

Reply:
After we do not update CPU used status, all network will use full CPU resource as previous release.